controller/src/main/java/io/pravega/controller/util/Config.java

```diff
@@ -125,14 +287,17 @@ public static GRPCServerConfig getGRPCServerConfig() {
                 .build();
     }
 
-    public static MetricsConfig getMetricsConfig() {
+    private static MetricsConfig createMetricsConfig(Properties p) {
```

Comment:
If i remember correctly, we needed this method because we had our metrics in the config with a path prefix. We could use the same metrics name as used in segment store